src/block/block_compact.c

```diff
@@ -442,10 +442,12 @@ __wt_block_compact_progress(WT_SESSION_IMPL *session, WT_BLOCK *block)
               100);
             __wt_verbose_debug1(session, WT_VERB_COMPACT_PROGRESS,
               "compacting %s for %" PRIu64 " seconds; reviewed %" PRIu64
-              " pages, rewritten %" PRIu64 " pages (%" PRIu64 "MB), approx. %d%% done",
+              " pages, rewritten %" PRIu64 " pages (%" PRIu64
+              "MB). Approx. %d%% of the estimated work done.%s",
               block->name, time_diff_start, block->compact_pages_reviewed,
               block->compact_pages_rewritten, block->compact_bytes_rewritten / WT_MEGABYTE,
-              progress);
+              progress,
+              progress == 100 ? " More work has been discovered since the estimation." : "");
```

Comment:
Is it possible or worthwhile to call the estimation function again at this point?

Reply:
@pmacko86 what do you think of this?

Reply:
I have created [WT-12319](https://jira.mongodb.org/browse/WT-12319). @pmacko86, please can you share your opinion in JIRA?